concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2106,9 +2106,9 @@ jobs:
                       OAUTH_CLIENT_SECRET: "(( grab uaa_clients_paas_admin_secret ))"
                       API_URL: "https://api.${SYSTEM_DNS_ZONE_NAME}"
                       BILLING_URL: "https://billing.${SYSTEM_DNS_ZONE_NAME}"
-                      UAA_URL: "https://uaa.${SYSTEM_DNS_ZONE_NAME}"
                       NOTIFY_API_KEY: "${NOTIFY_API_KEY}"
                       NOTIFY_WELCOME_TEMPLATE_ID: "${NOTIFY_WELCOME_TEMPLATE_ID}"
+                      SESSION_SECRET: "session-(( grab uaa_clients_paas_admin_secret ))"
```

Comment:
Can you explain (in the commit message) what this change will do?

Will it invalidate people's existing sessions?

If it's important that this is set, should we prevent `paas-admin` from starting when it's not?